### PR TITLE
Feature/new/add switch and skip controls

### PR DESCRIPTION
**Pull request by wouterbin**
Now we left the custom HTML5 video controls on the frontend project. We need the controls for switching media and skipping back and forward in this project.

Note: The styling of the controls should be fixed at some point. We need a front-ender for this (@tinux ?).

### CODE REVIEW

**Comment by kjwierenga** [comment]
Enable CodeShip CI integration?

**Review by kjwierenga — APPROVED** [review]
Hard to judge with my limited Javascript knowledge, but looks ok. Do the tests pass? May add CodeShip integration.

**Review by tinux — APPROVED** [review]
wow, a lot of code to glue it all together, but inevitable, nice job. 
I 'll check what I can do regarding the styling of the buttons.

File: addon/components/ivy-videojs-player.js [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+    return `vjs-switch-media-button ${super.buildCSSClass()}`;
+  }
+}
+
+class SkipForwardButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+  }
+  buildCSSClass() {
+    return `vjs-skip-forward icon pref10 ${super.buildCSSClass()}`;
```
Comment (kjwierenga) [inline addon/components/ivy-videojs-player.js:20]: Assuming you mean previous, `pref10` should be `prev10`? Why is the number 10 hardcoded here? Why is this class needed when you already have `vjs-skip-forward`?

Looking at it now, it is even stranger... Why have class `pref10` on a skip forward button? That's the opposite direction...

File: addon/components/ivy-videojs-player.js [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+  }
+}
+
+class SkipBackwardButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+  }
+  buildCSSClass() {
+    return `vjs-skip-back icon next10 ${super.buildCSSClass()}`;
```
Comment (kjwierenga) [inline addon/components/ivy-videojs-player.js:29]: Again, why class `next10` on the skip backward button?

**Review by kjwierenga — COMMENTED** [review]
May check and correct the css class names?

File: addon/components/ivy-videojs-player.js (hunk 1 above)
Comment (wouterbin) [reply addon/components/ivy-videojs-player.js:20]: As usual everything has a reason. The reason here is that the classes "icon pref10" add a custom icon. This is how it was delivered by Inpoint. You will not find the css class here its in the frontend.

vjs-skip-forward is an videojs class.

**Pull request merged**